modules/tensor_mechanics/examples/coal_mining/cosserat_elastic.i

```diff
@@ -0,0 +1,569 @@
+# Strata deformation and fracturing around a coal mine
+#
+# A 2D geometry is used that simulates a transverse section of
+# the coal mine.  The model is actually 3D, but the "x"
+# dimension is only 10m long, meshed with 1 element, and
```

Comment:
Most of the plasticity models that i intend on exploring in this example are only formulated in 3D (DruckerPrager and MohrCoulomb).  I wanted to keep the same setup throughout all the simulations.